### DISCUSSION TITLE
第5节课作业(18班)

### DISCUSSION THREAD

**Discussion by mattheliu** [InternLM/Tutorial#153] [作业]
请提交作业的链接
Please submit a link to the assignment.

**Comment by yestolife** [comment]
<img width="960" alt="1" src="https://github.com/InternLM/tutorial/assets/34405348/64c6f675-b203-49cb-a5a8-2c5a11cce5c8">

**Comment by Heitao5200** [comment]
https://github.com/Heitao5200/my_blog/blob/master/docs/%E7%BB%84%E9%98%9F%E5%AD%A6%E4%B9%A0/%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%AE%9E%E6%88%98/TASK5.md

**Comment by 2001wjh** [comment]
https://www.yuque.com/wjh2001/rdvfen/cc5sohlqb7sb4r4n?singleDoc# 《LMDeploy 大模型量化部署实践（作业）》

**Comment by DrZhang1208** [comment]
[第五次作业](https://blog.csdn.net/qq_42866033/article/details/135631497?spm=1001.2014.3001.5501)

**Comment by tackhwa** [comment]
[第5节课笔记+笔记](https://nutritious-chill-7a9.notion.site/LMDeploy-a9422127d8824298841a54140231a2cd?pvs=4)

**Comment by ggfengxuan** [comment]
[Ⅴ.LMDeploy 的量化和部署作业](https://v1y50l9n58a.feishu.cn/wiki/TmCrwmrHvinp5UksBE4cno10n8b?from=from_copylink)

**Comment by xwhclaire** [comment]
[Assignment 5 LMDeploy大模型量化部署](https://github.com/xwhclaire/StudyPackages/blob/main/LLM%20training/Assignment%205%20LMDeploly%E5%A4%A7%E6%A8%A1%E5%9E%8B%E9%87%8F%E5%8C%96%E9%83%A8%E7%BD%B2%E5%AE%9E%E9%AA%8C.md)

**Comment by scorpionfree98** [comment]
[第五节课笔记+作业](http://t.csdnimg.cn/TDnvn)

**Comment by ZMcc18** [comment]
https://www.yuque.com/zixul/avxnzn/fwttpmeg4niheegy?singleDoc# 《lesson5 LMDeploy 的量化和部署 作业&笔记》

**Comment by mattheliu** [comment]
https://github.com/mattheliu/gitblog/issues/5
